docker/testrunner/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y sudo openjdk-7-jdk xorg xvfb xserver-xorg-video-dummy
+RUN useradd -m -G users testrunner
+ADD entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
Again, sorry for such a late answer.
Regarding the permissions, I wanted to make sure the file got the right permissions.
I wasn't aware it would be useless since it is executed during the building process.